AprilTagTrackers/Connection.cpp

```diff
@@ -50,7 +50,7 @@ void Connection::Connect()
     if (word != "numtrackers")
     {
         wxMessageDialog dial(NULL,
-            wxT("Could not connect to SteamVR driver. Make sure SteamVR is running and the apriltagtrackers driver is installed. \nYou may also have to run bin/ApriltagTrackers.exe as administrator, if error code is not 2. \nError code: " + std::to_string(GetLastError())), wxT("Error"), wxOK | wxICON_ERROR);
+            wxT("Could not connect to SteamVR driver. Make sure SteamVR is running and the apriltagtrackers driver is installed. \nYou may also have to run bin/ApriltagTrackers.exe as administrator, if error code is not 2. \nError code: " ), wxT("Error"), wxOK | wxICON_ERROR);

```

Comment:
This message needs to be updated so that it doesn't refer to `ApriltagTrackers.exe` on linux. It should also not say `Error code: ` without an error code.

Reply:
I removed that, since there is no function equivalent, and I was planning on replacing it. However, I believe @ju1ce told me on Discord that this if statement doesn't even do anything, so it could just be removed?

Reply:
I think the method for getting the error will become apparent once we get to testing and implementing unix sockets.